04_setup_ironic.sh

```diff
@@ -17,11 +18,18 @@ for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE IPA_DOWNLOADER_IMAGE COREOS
         # Clone to ~ if not there already
         [ -e "$REPOPATH" ] || git clone $IMAGE $REPOPATH
         cd $REPOPATH
-        export $IMAGE_VAR=localhost/${IMAGE##*/}:latest
+        export $IMAGE_VAR=${IMAGE##*/}:latest
         sudo podman build -t ${!IMAGE_VAR} .
         cd -
-    else
-        sudo podman pull "$IMAGE"
+        sudo podman push --tls-verify=false ${!IMAGE_VAR} localhost:5000/localimages/${!IMAGE_VAR}
+        export $IMAGE_VAR=192.168.111.1:5000/localimages/${!IMAGE_VAR}
+
+        clone_installer
```

Comment:
We don't need to clone_installer here since that moved to `03_build_installer.sh` and `KNI_INSTALL_FROM_GIT=true` is set in common.sh below.